k2/csrc/util/loguru/loguru.cc

```diff
@@ -0,0 +1,1894 @@
+/*
```

Comment:
I suggest downloading these two files via cmake. You can refer to k2/cmake for how pybind11 is downloaded and included into k2.